src/main/java/com/app/katchup/Meeting/MeetingService.java

```diff
@@ -11,11 +11,14 @@
 public class MeetingService {
     @Autowired
     MeetingRepository meetingRepository;
-    public void createMeeting(Meeting meeting){
-        if(meetingRepository.findIfHostHasDuplicateMeeting(meeting.getHost(),meeting.getStartDateTime(),meeting.getEndDateTime(),
+    public Meeting createMeeting(Meeting meeting){
+        if(meetingRepository.findMeetingByFilter(meeting.getHost(),
+                meeting.getStartDateTime(),meeting.getEndDateTime(),
                 meeting.getVenue())==null){
             meetingRepository.save(meeting);
         }
 
+        return meeting;
```

Comment:
if the condition is true, save it to the meeting and get the same meeting from DB then only the user will know the meeting-id and meeting pwd. 
if condition is false, return null

Reply:
Another option, put the application in debugger and show me the pwd and meeting id without getting the same meeting from DB. If this is possible, you dont need to do the DB fetching again.